### ISSUE TITLE
Plugin apparently stalled

### ISSUE THREAD

**Issue by pcav** [antarctica/oceancolor_downloader#37]
I can no longer download anything from the plugin. See attached image.

![nasa_ocean_downloader](https://cloud.githubusercontent.com/assets/691842/22302303/742bb312-e32e-11e6-8883-fa4b5d69ad62.png)

**Comment by pcav** [comment]
I tried again: it takes a huge time, but it gets to an end. The problem is that apparently nothing is downloaded, and of course not loaded.

**Comment by lo-ise** [comment]
Hi @pcav, sorry for the delay. I haven't had a lot of time to check this out. If anyone has an idea of what might be causing this, please create a comment. This may also be related to issue #36.

**Comment by pcav** [comment]
Sorry, no idea. Maybe you can ask on qgis-dev.

**Comment by janitzio59** [comment]
Yes, indeed, is the same problem as #36, I get the same box telling it is preparng to download, and begins to download, but... nothing!
